### PR TITLE
Date/Time working, Description working, Alert given on event post and

### PR DESCRIPTION
**Pull request by MorrowCameron**
redirected to browse event page, Date and time working, description added with growing description box.

### CODE REVIEW

File: Eat_and_Greet/src/screens/PostEvent/index.tsx [hunk 1]
```diff
@@ -62,14 +72,17 @@ export default function PostEvent() {
           let uid : number = uidList[0] + 1
           await setDoc(doc(db, "Events", uid.toString()), {
             Category: category,
-            Date: eventDate,
-            EndTime: endTime,
+            Date: dateOfEvent.toDateString(),
+            EndTime: endTime.toTimeString().split(' ')[0],
```
Comment (MorrowCameron) [inline Eat_and_Greet/src/screens/PostEvent/index.tsx:76]: Split is to remove annoying time zone clutter

**Review by Logidoke — APPROVED** [review]
LTGM! (Let's Get This Money)

File: Eat_and_Greet/src/comps/GrowingTextbox/textbox.tsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
+import React, { Component } from 'react';
+import { TextInput} from 'react-native';
+
+export default class GrowingTextbox extends Component {
```
Comment (R303Vermeulen) [inline Eat_and_Greet/src/comps/GrowingTextbox/textbox.tsx:4]: woah, a class. this is uncharted territory. Good shit though

File: Eat_and_Greet/src/comps/GrowingTextbox/textbox.tsx [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+    return (
+    <TextInput
+      placeholder="Your Placeholder"
+      onChangeText={(value) => this.handleText(value)}
+      style={newStyle}
+      editable={true}
+      multiline={true}
+      value={value}
+      onContentSizeChange={(e) => this.updateSize(e.nativeEvent.contentSize.height)}
```
Comment (R303Vermeulen) [inline Eat_and_Greet/src/comps/GrowingTextbox/textbox.tsx:40]: Not a review but curiousity, How does this get called? Everytime the text changes, or when it starts a new line? Either way well done

File: Eat_and_Greet/src/comps/GrowingTextbox/textbox.tsx (hunk 2 above)
Comment (MorrowCameron) [reply Eat_and_Greet/src/comps/GrowingTextbox/textbox.tsx:4]: Don't worry its copy pasted from stack overflow

File: Eat_and_Greet/src/comps/GrowingTextbox/textbox.tsx (hunk 3 above)
Comment (MorrowCameron) [reply Eat_and_Greet/src/comps/GrowingTextbox/textbox.tsx:40]: I believe its new line, its just a parameter surprisingly enough!

File: Eat_and_Greet/src/screens/PostEvent/index.tsx [hunk 4]
```diff
@@ -113,32 +139,43 @@ export default function PostEvent() {
...
+        />
         </View>
-      </SafeAreaView>
-      <RoundedButton name="Post Event" top="-2%" onPress={addPostToDB} />
+      </KeyboardAwareScrollView>
+      </View>
+      <RoundedButton name="Post Event" onPress={addPostToDB} />
       <Toolbar />
-    </SafeAreaView>
+    </View>
```
Comment (R303Vermeulen) [inline Eat_and_Greet/src/screens/PostEvent/index.tsx:179]: Curious why you went from SafeAreaView to regular View? 
Prolly doesn't matter with scroll view anyway but curious

File: Eat_and_Greet/src/screens/PostEvent/index.styles.tsx [hunk 5]
```diff
@@ -1,39 +1,41 @@
...
 const postStyle = StyleSheet.create({
   InfoContainer: {
     backgroundColor: "#8EA7E9",
-    height: "80%",
     width: "87%",
-    top: "-5%",
     borderRadius: 10,
     justifyItems: "space-between",
-    alignItems: "center",
+    marginVertical: 10,
```
Comment (R303Vermeulen) [inline Eat_and_Greet/src/screens/PostEvent/index.styles.tsx:12]: What's the difference between this and margin-top?

**Review by R303Vermeulen — APPROVED** [review]
Asked some questions for clarity but looks good! Great work with the growing textbox and KeyboardAwareScrollView

**Pull request merged**